recipes-sota/ostree/ostree_git.bb

```diff
@@ -1,5 +1,6 @@
 SUMMARY = "Tool for managing bootable, immutable, versioned filesystem trees"
-LICENSE = "GPLv2+"
+HOMEPAGE = "https://ostree.readthedocs.io/en/latest/"
+LICENSE = "LGPLv2+"
 LIC_FILES_CHKSUM = "file://COPYING;md5=5f30f0716dfdd0d91eb439ebec522ec2"
 
 inherit autotools-brokensep pkgconfig systemd bash-completion gobject-introspection
```

Comment:
Minor question: is `autotools-brokensep` still necessary here? It's no longer required for rocko and later versions.

Reply:
Are you sure? It looks like it's being used in rocko branch of poky in several places.

Reply:
Our rocko version of this recipe doesn't use it, though, that's all.

Reply:
It seems that replacing autotools-brokensep with autotools is OK for morty. I've rebased the ostree patch locally. Let me spin a couple of builds from scratch more to double check and I will provide further information.

Thanks, Leon